pulsar-broker/src/main/java/org/apache/pulsar/broker/intercept/BrokerInterceptors.java

```diff
@@ -102,6 +108,78 @@ public void beforeSendMessage(Subscription subscription,
         }
     }
 
+    @Override
+    public void consumerCreated(ServerCnx cnx,
+                                 Consumer consumer,
+                                 Map<String, String> metadata) {
+        for (BrokerInterceptorWithClassLoader value : interceptors.values()) {
```

Comment:
I know this was already the style of other methods here, though we should try to avoid the allocation of an iterator instance in the case where there are no interceptors. eg: 

```java
if (interceptors.isEmpty()) {
  return;
}

for (BrokerInterceptorWithClassLoader value : interceptors.values()) {
     // .... 
}
```

Reply:
+1

Reply:
+1 please check null or empty first.

Reply:
Done